dmci/mmd_tools/check_mmd.py

```diff
@@ -192,38 +210,40 @@ def full_check(doc): # pragma: no cover
             logger.info('OK - geographic_extent/rectangle')
         else:
             logger.info('NOK - geographic_extent/rectangle -> check debug log')
-        valid = valid and rect_ok
+        valid &= rect_ok
     else:
         logger.debug('No geographic_extent/rectangle element.')
 
     # Check that cf name provided exist in reference Standard Name Table
-    cf_elements = doc.findall('./{*}keywords[@vocabulary="Climate and Forecast Standard Names"]')
-    if len(cf_elements) == 1:
-        logger.debug('Checking elements keyword from vocabulary CF ...')
-        cf_list = [elem.text for elem in cf_elements[0]]
-        if len(cf_list) > 1:
-            logger.info(f'NOK - CF names -> only one CF name should be provided - {cf_list}')
-            valid = False
-        # Check CF names even if more than one provided
-        cf_ok = check_cf(cf_list)
-        if cf_ok:
-            logger.info('OK - CF names')
-        else:
-            logger.info('NOK - CF names -> check debug log')
-        valid = valid and cf_ok
-    elif len(cf_elements) > 1:
-        valid = False
-        logger.debug('NOK - More than one element with keywords[@vocabulary="Climate and '
-                     'Forecast Standard Names"]')
-    else:
-        logger.debug('No CF standard names element.')
+    # cf_elements = doc.findall('./{*}keywords[@vocabulary="Climate and Forecast Standard Names"]')
+    # if len(cf_elements) == 1:
+    #     logger.debug('Checking elements keyword from vocabulary CF ...')
+    #     cf_list = [elem.text for elem in cf_elements[0]]
+    #     if len(cf_list) > 1:
+    #         logger.info(f'NOK - CF names -> only one CF name should be provided - {cf_list}')
+    #         valid = False
+    #     # Check CF names even if more than one provided
+    #     cf_ok = check_cf(cf_list)
+    #     if cf_ok:
+    #         logger.info('OK - CF names')
+    #     else:
+    #         logger.info('NOK - CF names -> check debug log')
+    #     valid &= cf_ok
+    # elif len(cf_elements) > 1:
+    #     valid = False
+    #     logger.debug(
+    #         'NOK - More than one element with keywords[@vocabulary="Climate and '
+    #         'Forecast Standard Names"]'
+    #     )
+    # else:
+    #     logger.debug('No CF standard names element.')
 
     # Check controlled vocabularies
-    voc_ok = check_vocabulary(doc)
-    valid = valid and voc_ok
-    if voc_ok:
-        logger.info('OK - Controlled vocabularies.')
-    else:
-        logger.info('NOK - Controlled vocabularies -> check debug log')
+    # voc_ok = check_vocabulary(doc)
+    # valid &= voc_ok
+    # if voc_ok:
+    #     logger.info('OK - Controlled vocabularies.')
+    # else:
+    #     logger.info('NOK - Controlled vocabularies -> check debug log')
```

Comment:
why are we not testing this?

Reply:
Because the vocabulary checks don't work.